<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExecSqlWidget</class>
 <widget class="QWidget" name="ExecSqlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
/*背景色*/
	background-color: rgb(210, 220, 255);
/*边框样式*/
border-style: outset;
/*边框宽度为4像素*/
border-width: 4px;
/*边框圆角半径*/
border-radius: 10px;
/*边框颜色*/
border-color: rgba(255, 225, 255, 30);
/*字体*/
font: bold 20px;
/*字体颜色*/
color: rgb(35, 35, 35);
/*填衬*/
padding: 6px;
}

QPushButton:disabled{
background-color: rgba(211, 211, 211, 180);
color: rgba(39, 39, 39, 100);
}

/*鼠标悬停在按钮上时*/
QPushButton:hover{
	background-color: rgb(102, 102, 255,200);
border-color: rgba(255, 225, 255, 200);
color:rgba(0, 0, 0, 200);
}

/*按钮被按下时*/
QPushButton:pressed {
background-color: rgba(102, 102, 255, 200);
border-color: rgba(255, 225, 255, 30);
border-style: inset;
color:rgba(0, 0, 0, 100);
}

/*********标签*************/
QLabel{
	font: bold 17px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,0">
     <item>
      <widget class="QTextEdit" name="textE_sqlScript">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tWidget_showResults"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0,0">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pBtn_refresh">
         <property name="text">
          <string>刷新数据</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pBtn_showResults">
         <property name="text">
          <string>显示结果</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pBtn_saveToFile">
         <property name="text">
          <string>将结果存到文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pBtn_saveSqlScript">
         <property name="text">
          <string>保存SQL脚本</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
